src/index.js

```diff
@@ -1,59 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import Payment from 'payment';
+import creditCardType from 'credit-card-type';
+import luhn from 'luhn';
+
+import { configure } from './config';
+import { sanitizeNumber } from './utils';
 
 class ReactCreditCards extends React.Component {
   constructor(props) {
     super(props);
 
-    this.setCards();
+    const validCardTypes = configure();
```

Comment:
I decided to remove the config from here and put on its own file so we don't pollute the constructor and it's easier to change when needed